cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Partition.java

```diff
@@ -138,10 +138,20 @@ public void swapFollowerPositions(int followerIndex, int otherFollowerIndex) {
     Set<Broker> partitionBrokers = new HashSet<>();
     // Add leader and follower brokers.
     partitionBrokers.add(_leader.broker());
-    partitionBrokers.addAll(followerBrokers());
+    _followers.forEach(r -> partitionBrokers.add(r.broker()));
```

Comment:
What does this change provide?

Reply:
Avoid creation of another list and copying it over.